Description: fix POD spelling
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=74239
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=74239
Author: Jonathan Yu <jawnsy@cpan.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2012-01-22

--- a/lib/Net/Amazon/EC2.pm
+++ b/lib/Net/Amazon/EC2.pm
@@ -420,7 +420,7 @@
 
 The name of the group to add security rules to.
 
-=item SourceSecurityGroupName (requred when authorizing a user and group together)
+=item SourceSecurityGroupName (required when authorizing a user and group together)
 
 Name of the group to add access for.
 
@@ -3418,7 +3418,7 @@
 
 The name of the group to revoke security rules from.
 
-=item SourceSecurityGroupName (requred when revoking a user and group together)
+=item SourceSecurityGroupName (required when revoking a user and group together)
 
 Name of the group to revoke access from.
 
